win32ss/printing/base/winspool/printers.c

```diff
@@ -449,9 +449,17 @@ EnumPrintersA(DWORD Flags, PSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cbB
  
     /* Ref: https://stackoverflow.com/questions/41147180/why-enumprintersa-and-enumprintersw-request-the-same-amount-of-memory */
     bReturnValue = EnumPrintersW(Flags, pwszName, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
-    HeapFree(hProcessHeap, 0, pwszName);
+    TRACE("*pcReturned is '%d' and bReturnValue is '%d' and GetLastError is '%ld'.\n",
+          *pcReturned, bReturnValue, GetLastError());
 
-    TRACE("*pcReturned is '%d' and bReturnValue is '%d' and GetLastError is '%ld'.\n", *pcReturned, bReturnValue, GetLastError());
+    if (pwszName)
+        HeapFree(hProcessHeap, 0, pwszName);
```

Comment:
Add braces please.
This is okay since you're changing something else right next to it (or in new code), but for future reference, HeapFree accepts null pointers and it's not necessary to add null checks to existing code.

Reply:
> HeapFree()

Done. I know. I simply tried to follow this file style...